misc/plot-ampliconstats

```diff
@@ -1250,3 +1249,97 @@ END
 
     $combined_done = 1;
 }
+
+#---------------------------------
+#adding html file to show plots
+my $fname  = "index.html";
+
+open(my $fh,'>',$fname) or error("$fname: $!");
+print $fh q[
+<html>
+<head>
+<style>
+.thumbnail
+{
+    text-decoration:none;
+    color:black;
+    font-weight:bold;
+}
+.thumbnail span  /*CSS for enlarged image*/
+{
+    visibility: hidden;
+    position: absolute;
+    padding: 5px;
+    border: 1px solid #000;
+    background-color: #e5e5e5;
+}
+.thumbnail:hover span  /*CSS for enlarged image on hover*/
+{
+    visibility: visible;
+    left: 550px;
+    top: 10px;
+}
```

Comment:
This bit of CSS looks redundant at the moment.  It looks like it got copied over from `plot-bamstats`.  Did you want to be able to see a bigger version when hovering over the thumbnail?

To get the feature to work, you would need to add a `<span>` containing the full-size version of each image next to the thumbnail, something like this:
```
print $fh qq[<td><a class="thumbnail" href="$imgs[$i]"><img src="$imgs[$i]" width="200px"><span><img src="$imgs[$i]"></span></a></td>\n];
```
You might also want to look into where the bigger image pops up.  At the moment it would obscure some of the thumbnails, making them difficult to select.

If you don't want the bigger image on hover (and don't want to add it later) it would be good to remove these lines to avoid confusion.

Reply:
On discussing with @jkbonfield we decided that the behaviour of popping up full-sized images wasn't going to work well for `plot-ampliconstats` output, so I've added a commit to remove this bit of CSS.  I've also got rid of the `.pad` and `.right` sections which weren't referenced anywhere.

I'll squash the commits together on merge and rebase #1299 on top.